include/igl/copyleft/comiso/nrosy.cpp

```diff
@@ -297,9 +294,18 @@ void igl::copyleft::comiso::NRosyField::prepareSystemMatrix(const int N)
     if (!isFixed_i)
     {
       row = tag_t[i];
-      if (isFixed_i) b(row) += -2               * hard[i]; else T.push_back(Eigen::Triplet<double>(row,tag_t[i]  , 2             ));
-      if (isFixed_j) b(row) +=  2               * hard[j]; else T.push_back(Eigen::Triplet<double>(row,tag_t[j]  ,-2             ));
-      if (isFixed_p) b(row) += -((4 * igl::PI)/Nd) * p[eid] ; else T.push_back(Eigen::Triplet<double>(row,tag_p[eid],((4 * igl::PI)/Nd)));
+      if (isFixed_i)
```

Comment:
If you're doing some (clearly much needed!) cleanup, do you mind adding some braces on these if statements? No brace if statements can introduce nasty bugs.

Reply:
This is not in the style guideline and we have plenty of existing code that is not using braces. Even though I agree with you on the potential pitfalls of not using braces, I think this is not needed here.

Reply:
Fine, we don't have style guidelines for this. I think we should but that's a separate issue. @copyme, feel free to not add braces.